Dockerfiles/base/00.python-redhat-ubi.Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM registry.access.redhat.com/ubi8/python-39
+FROM registry.access.redhat.com/ubi8-minimal
```

Comment:
Thoughts on renaming this file from `00.python-redhat-ubi.Dockerfile` to `00.redhat-ubi.Dockerfile` since the base is no longer `ubi8-minimal`?

Reply:
I start with ubi-minimal bit the first thing i do it install python 3.8 ... so the container (which gets its name from the file name) still results with python on redhat ubi.  Same as before the lastest change.

Reply:
I think we should keep the name `python-redhat-ubi`. Even though we are not using `ubi8/python-39`, we are still installing python ourselves. So technically this image is still a python-redhat-ubi

Reply:
Good point!